papermill/cli.py

```diff
@@ -28,11 +59,10 @@ def papermill(notebook, output, parameters, raw_parameters, parameters_file, par
         parameters_final = {}
         for name, value in parameters:
             parameters_final[name] = _resolve_type(value)
-        for name, value in raw_parameters:
+        for name, value in parameters_raw:
             parameters_final[name] = value
 
-    # Notebook Execution
-    execute_notebook(notebook, output, parameters_final, kernel_name=kernel)
+    execute_notebook(notebook_path, output_path, parameters_final, kernel_name=kernel)
```

Comment:
...and pass `src`, `dst` in directly here as `execute_notebook(src, dst, parameters_final, kernel_name=kernel)` or even as `execute_notebook(notebook_path=src, output_path=dst, parameters_final, kernel_name=kernel)`

Reply:
I went ahead and merged, you're welcome to make these changes in another PR. I wanted to make sure we brought in what you've already done.